crypto/lhash/lhash.c

```diff
@@ -42,7 +42,23 @@ static int expand(OPENSSL_LHASH *lh);
 static void contract(OPENSSL_LHASH *lh);
 static OPENSSL_LH_NODE **getrn(OPENSSL_LHASH *lh, const void *data, unsigned long *rhash);
 
+static void set_error(OPENSSL_LHASH *lh)
+{
+    CRYPTO_atomic_write(&lh->err.value, 1, lh->err.lock);
+}
+
+static void clear_error(OPENSSL_LHASH *lh)
+{
+    CRYPTO_atomic_write(&lh->err.value, 0, lh->err.lock);
+}
+
 OPENSSL_LHASH *OPENSSL_LH_new(OPENSSL_LH_HASHFUNC h, OPENSSL_LH_COMPFUNC c)
+{
+    return OPENSSL_LH_new_ex(h, c, 0L);
```

Comment:
So I wonder if this should be called with 0 or OPENSSL_LH_FLAGS_STATS

Reply:
We want to move to default without stats, but we can't break current behavior in this release.